vhpc_toolkit/config_objs.py

```diff
@@ -1229,6 +1233,8 @@ def create_dvs(self, host_vmnics, dvs_name, num_uplinks=4):
             vim.dvs.VmwareDistributedVirtualSwitch.LacpApiVersion.multipleLag
         )
         dvs_config_spec.numStandalonePorts = num_uplinks
+        if max_mtu:
+            dvs_config_spec.maxMtu = max_mtu
```

Comment:
It seems that the API use "maxMtu"?
In networking, MTU means maximum transmission unit.

Reply:
I believe that's a mistake